wsl-vpnkit

```diff
@@ -11,24 +11,30 @@ VPNKIT_STORE="/files/vpnkit/vpnkit.exe"
 NPIPERELAY_STORE="/files/npiperelay/npiperelay.exe"
 VPNKIT_PATH="$USERPROFILE/wsl-vpnkit/wsl-vpnkit.exe"
 NPIPERELAY_PATH="$USERPROFILE/wsl-vpnkit/npiperelay.exe"
-TAP_NAME="eth1"
-VPNKIT_GATEWAY_IP="192.168.67.1"
-VPNKIT_HOST_IP="192.168.67.2"
-VPNKIT_LOWEST_IP="192.168.67.3"
-VPNKIT_HIGHEST_IP="192.168.67.14"
-VPNKIT_WSL2_IP="$VPNKIT_LOWEST_IP"
-VPNKIT_DEBUG=
-WSL2_GATEWAY_IP="$(cat /etc/resolv.conf | awk '/^nameserver/ {print $2}')"
-CHECK_HOST=example.com
-DNS_IP="$VPNKIT_GATEWAY_IP"
-CHECK_DNS=1.1.1.1
 
 echo "starting wsl-vpnkit"
+
+# Load defaults and user config if needed
+. /app/defaults.conf
```

Comment:
We should do a check if this default config file exists before trying to load it.

Reply:
✔